### PR TITLE
Feat: Translate toast messages by adding msgi18n to callbacks

### LINKED ISSUE #3259: Add localisation (i18n) to the backend alerts and responses
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

You should add translations for   alerts(their logic is in the backend):
callback({
    ok: true,
    msg: "Saved.",
    monitorID: bean.id,
});
P.S. Code taken from server.js

### ✔️ Solution

i don't know

### ❓ Alternatives

_No response_

### 📝 Additional Context

_No response_

### PR DESCRIPTION
**Pull request by chakflying**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description

Fixes #3259

I thought about adding a serverside translation library (like `i18next`), but each library has subtly different ways of handling things like regional fallbacks, interpolation etc. It's just not worth introducing another library to handle this. Also, if there exist an API for the server, we would not expect the returned messages to be translated. Hence I think we should focus on internationalization being a frontend problem only. But on the other hand, we may not want the frontend to display translated messages just because a key happens to be available. Therefore I arrived at this solution.

TODO: Add `msgi18n` to all translatible `msg`, add corresponding keys to `en.json`.

## Type of change

- User interface (UI)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [ ] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

**Comment by DevItq** [comment]
@chakflying I think this will be helpful: https://phrase.com/blog/posts/node-js-i18n-guide/

**Comment by chakflying** [comment]
I have already explained in the description why adding another library is not ideal in this situation.

File: src/lang/hr-HR.json [hunk 1]
```diff
@@ -348,7 +348,7 @@
     "Discard": "Odbaci",
     "Cancel": "Otkaži",
     "Powered by": "Pokreće",
-    "Saved": "Spremljeno",
+    "Saved.": "Spremljeno.",
```
Comment (CommanderStorm) [inline src/lang/hr-HR.json:351]: We are inconsistent between `Saved.` and `Saved`. 
Please unify this to one message or add both keys to the translations

Comment (chakflying) [reply src/lang/hr-HR.json:351]: Fixed.

**Comment by louislam** [comment]
I think your approach is a good start, may still need to think how to handle some longer sentences with dynamic values though.

**Comment by chakflying** [comment]
revised the format to support interpolation: 
```javascript
{
  msg: {
    key: "clearDataOlderThan",
    values: [
      "30"
    ]
  }, // or
  msg: "Saved."
  msgi18n: true
}
```

**Pull request merged**